docs/channels.md

```diff
@@ -0,0 +1,768 @@
+
+## Using %channels
+Code examples here are dojo commands.
+
+### Scries
+#### /channels
+```hoon
+=c -build-file /=groups=/sur/channel/hoon
+.^(channels:c %gx /=channels=/channels/channel-channels)
+```
+
+Get the channels our ship is in and their contents. Returns `channels`
+
+[channels](#channels-1)
+#### /unreads
+```hoon
+=c -build-file /=groups=/sur/channel/hoon
+.^(unreads:c %gx /=channels=/unreads/channel-unreads)
+```
+
+Get unread information for the channels our ship is in. Returns `unreads`
+
+[unreads](#unreads-2)
+#### /init
+```hoon
+=c -build-file /=groups=/sur/channel/hoon
+.^([unreads:c channels:c] %gx /=channels=/init/noun)
+```
+
+Combination of `/channels` and `/unreads`. Returns `[unreads channels]`
+
+[unreads](#unreads-2) \| [channels](#channels-1)
+### Pokes
+#### %channel-action
+
+Takes an [a-channels](#a-channels), which can be one of several actions. Each action is covered below.
+
+##### %create
+```hoon
+=c -build-file /=groups=/sur/channel/hoon
+=g -build-file /=groups=/sur/groups/hoon
+
+=create-channel :*
+  %chat
+  %mychat
+  [our %mygroup]
+  title='My chat channel'
+  description='Chat about things'
+  readers=~
+  writers=~
+==
+
+:channels &channel-action [%create create-channel]
+```
+
+Create a channel. This also updates the metadata in the %groups agent.
+
+[create-channel](#create-channel)
+
+##### %channel
+[nest](#nest)
+
+###### %join
+```hoon
+=nest [%chat ~palnet-sampel %mychat]
+=action [%join group=[~sampel-palnet %mygroup]]
+
+:channels &channel-action [%channel nest action]
+```
+
+Join a channel. In this example, ~sampel-palnet is the group host and ~palnet-sampel is the channel host.
+###### %leave
+```hoon
+=nest [%chat ~palnet-sampel %mychat]
+=action [%leave ~]
+
+:channels &channel-action [%channel nest action]
+```
+
+Leave a channel
+###### %read
+```hoon
+=nest [%chat our %mychat]
+=action [%read ~]
+
+:channels &channel-action [%channel nest action]
+```
+
+Mark a channel as read.
+###### %read-at
+```hoon
+=nest [%chat our %mychat]
+=action [%read-at now]
+
+:channels &channel-action [%channel nest action]
+```
+
+Mark a channel as read up to a certain time.
+###### %watch
+```hoon
+=nest [%chat our %mychat]
+=action [%watch ~]
+
+:channels &channel-action [%channel nest action]
+```
+
+Watch a channel for unreads.
+###### %unwatch
+```hoon
+=nest [%chat our %mychat]
+=action [%unwatch ~]
+
+:channels &channel-action [%channel nest action]
+```
+
+Stop watching a channel for unreads.
+
+##### Actions
+All of the previous pokes affected your local ship (joining and leaving, unreads, and so on). The following pokes are *actions*, meaning they will be synced across the network. When `%channels` on your ship hears an action, it issues a *command* to the host's `%channels-server` (see the above [diagram](#diagram)).
```

Comment:
Should probably explain this ahead of time. But also, the previous ones were actions, too, by our type naming convention. Marking some as "affects local state only" is fair, but shouldn't be made to seem like a big deal, and certainly shouldn't conflate terminology (especially if we're not talking about `%channels-server`/`$command`s here).

The heading for this could be nicer, but per the above, I'd argue this doesn't need a separate heading/nesting level at all.

Reply:
Looking at the `/sur` file, I can see how this emphasis came about: the post & reply action types are defined in terms of their command types. Considering that the action types are the outward-facing ones, that should probably be the other way around. I may touch that up.

Reply:
@arthyn what do you think?